plugin.php

```diff
@@ -35,39 +37,91 @@ function queue_export( $config = null ) {
  * @param  string $config
  */
 function static_page_save( $config = null ) {
-
 	// Track the background task in an option for reporting / status checking.
 	$update_progress = [
 		'date'      => time(),
 		'urls'      => [],
 		'done_urls' => [],
 	];
+
 	$option_name = 'static_page_save_running';
 	update_option( $option_name, $update_progress );
 
-	$urls = get_site_urls();
+	$posts_per_page = 50;
 
-	$update_progress['urls'] = $urls;
-	update_option( $option_name, $update_progress );
+	$query_args = [
+		'post_type'              => 'any',
```

Comment:
Is post type `any` or `page` in the context of a static `page` process?

Reply:
Any.